core-web/tsconfig.base.json

```diff
@@ -55,7 +58,8 @@
             "@portlets/*": ["apps/dotcms-ui/src/app/portlets/*"],
             "@services/*": ["apps/dotcms-ui/src/app/api/services/*"],
             "@shared/*": ["apps/dotcms-ui/src/app/shared/*"],
-            "@tests/*": ["apps/dotcms-ui/src/app/test/*"]
+            "@tests/*": ["apps/dotcms-ui/src/app/test/*"],
+            "dot-languages": ["libs/portlets/dot-languages/portlet/src/index.ts"]
```

Comment:
Why exist 2 alias to the same index
`"@dotcms/portlets/dot-languages/portlet": [ "libs/portlets/dot-languages/portlet/src/index.ts" ],`
`"dot-languages": ["libs/portlets/dot-languages/portlet/src/index.ts"]`

Reply:
fixed